lib/src/model/product.dart

```diff
@@ -655,6 +670,11 @@ class Product extends JsonObject {
       this.ecoscoreGrade,
       this.ecoscoreScore,
       this.ecoscoreData,
+      this.dataQualityTags,
```

Comment:
Zero added value to put that kind of fields in the constructor. Please remove them if possible.